salt/cli/run.py

```diff
@@ -30,8 +30,9 @@ def run(self):
         try:
             if check_user(self.config["user"]):
                 pr = salt.utils.profile.activate_profile(profiling_enabled)
+                show_jid = self.config.get("show_jid", False)
                 try:
-                    ret = runner.run(full_return=True)
+                    ret = runner.run(full_return=True, show_jid=show_jid)
```

Comment:
why does this need to be passed as a kwarg? salt.runner has self.opts (the same as self.config here)

Reply:
@mattp- - thanks for the feedback. My intent in doing that was to ensure that the JID was *only* printed when the runner was called from `salt-run`. I'd assumed if I didn't do that, when called via the python api or netapi, that would be either returned or logged.. having tested with your suggested changes however, that doesn't seem to be the case.

I'm not sure where the stdout goes in those cases, but it doesn't seem to be an issue, so your much simpler solution is good - thanks!